docs/tutorial/t-deploy-kafka.md

```diff
@@ -0,0 +1,100 @@
+# Get a Charmed Kafka and Zookeeper up and running
+
+This is part of the [Charmed Kafka Tutorial](/t/). Please refer to this page for more information and the overview of the content. 
+
+## Deploy
+
+To deploy Charmed Kafka, all you need to do is run the following command, which will fetch the needed charms([Kafka](https://charmhub.io/kafka?channel=3/edge) and [Zookeeper](https://charmhub.io/zookeeper?channel=edge)) from [Charmhub](https://charmhub.io/) and deploy it to your model:
+
+```shell
+$ juju deploy zookeeper --channel latest/edge -n 5
+$ juju deploy kafka --channel 3/edge -n 3
+```
+
+After this, it is necessary to connect them:
+
+```shell
+$ juju relate kafka zookeeper
+```
+
+Juju will now fetch Charmed Kafka and Zookeeper and begin deploying it to the LXD cloud. This process can take several minutes depending on how provisioned (RAM, CPU,etc) your machine is. You can track the progress by running:
+```shell
+juju status --watch 1s
+```
+
+This command is useful for checking the status of Charmed Zookeeper and Charmed Kafka and gathering information about the machines hosting the two applications. Some of the helpful information it displays include IP addresses, ports, state, etc. 
+The command updates the status of the cluster every second and as the application starts you can watch the status and messages of Charmed Kafka and Zookeeper change. 
+
+Wait until the application is ready - when it is ready, `juju status --watch 1s` will show:
+```shell
+Model     Controller  Cloud/Region         Version  SLA          Timestamp
+tutorial  overlord    localhost/localhost  2.9.38   unsupported  08:20:12Z
+
+App        Version  Status  Scale  Charm      Channel      Rev  Exposed  Message
+kafka               active      3  kafka      3/edge       112  no       
+zookeeper           active      5  zookeeper  latest/edge   98  no       
```

Comment:
We could probably update the channel here and the revision

Reply:
Done